src/main/java/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.FileReader;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> listOfLines = new ArrayList<>();
+        File file = new File(fileName);
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(file))) {
```

Comment:
Use try-with-resources to ensure that the `BufferedReader` is closed properly. This is already done correctly here, so make sure to use this pattern consistently throughout your code.